features/support/sharedSteps.js

```diff
@@ -1,29 +1,61 @@
 const pactum = require('pactum');
 const { expression } = require('pactum-matchers');
-const {
-  Given, Then
-} = require('@cucumber/cucumber');
+const { Given, Then } = require('@cucumber/cucumber');
+const { includes } = require('pactum-matchers');
 
 // eslint-disable-next-line prefer-arrow-callback
 Given(/I set json body to the file at (.*)$/, function (fixture) {
   this.spec.withJson(fixture);
 });
 
-Then(/I expect response should validate against the profile (.*)$/, async function (profile) {
-  if (this.parameters.validatorServiceUrl === null) {
-    return Promise.resolve('skipped');
-  }
+Then(
+  /I expect response should validate against the profile (.*)$/,
+  async function (profile) {
+    if (this.parameters.validatorServiceUrl === null) {
+      return Promise.resolve('skipped');
+    }
+
+    const response = await pactum
+      .spec()
+      .post(`${this.parameters.validatorServiceUrl}?profile=${profile}`)
+      // eslint-disable-next-line no-underscore-dangle
+      .withJson(this.spec._response.json);
 
-  const response = await pactum.spec()
-    .post(`${this.parameters.validatorServiceUrl}?profile=${profile}`)
-    // eslint-disable-next-line no-underscore-dangle
-    .withJson(this.spec._response.json);
+    if (response.statusCode !== 200) {
+      Promise.reject(
+        new Error(
+          `Error executing FHIR validator service: ${response.statusMessage}`
+        )
+      );
+    }
 
-  if (response.statusCode !== 200) {
-    Promise.reject(new Error(`Error executing FHIR validator service: ${response.statusMessage}`));
+    pactum
+      .expect(response)
+      .to.have.jsonMatch(
+        'issue[*].severity',
+        expression(null, '!$V.includes("fatal")')
+      );
+    pactum
+      .expect(response)
+      .to.have.jsonMatch(
+        'issue[*].severity',
+        expression(null, '!$V.includes("error")')
+      );
+    return Promise.resolve();
   }
+);
 
-  pactum.expect(response).to.have.jsonMatch('issue[*].severity', expression(null, '!$V.includes("fatal")'));
-  pactum.expect(response).to.have.jsonMatch('issue[*].severity', expression(null, '!$V.includes("error")'));
-  return Promise.resolve();
-});
+// eslint-disable-next-line prefer-arrow-callback
+Then(
+  /I expect a response entry exists for each request entry (.*)/,
+  function (resources) {
+    const resourceList = resources.split(',').map((value) => value.trim());
+    const numResources = resourceList.length;
+    // eslint-disable-next-line no-plusplus
```

Comment:
In general, we don't want to disable linting rules unless there is no alternative. In this case, there is an alternative documented at https://eslint.org/docs/rules/no-plusplus.